scss/civicrm/common/_buttons.scss

```diff
@@ -338,13 +337,27 @@ $button-border-radius: 2px;
     }
   }
 
-  .crm-profile-selector-preview {
-    padding: 4px 12px;
+  .crm-profile-selector-preview,
+  .crm-profile-selector-edit,
+  .crm-profile-selector-copy,
+  .crm-profile-selector-create {
+    border: 0;
+    height: auto;
+    line-height: normal;
+    padding: 6px 12px;
 
     .crm-i {
       position: relative;
       top: -1px;
     }
+
+    &:not([disabled]) {
+      cursor: pointer;
+    }
+
+    &[disabled] {
+      opacity: 0.65;
```

Comment:
is this something we can make generic and put in a variable?

Reply:
@AkA84 The only other place where it is used is https://github.com/civicrm/org.civicrm.shoreditch/blob/master/base/scss/vendor/bootstrap/_buttons.scss#L50
which gets extended to other buttons already.

These 4 are the only buttons which does not have that so, I have added it as it is. When have more than one instance we will go for variable.